src/java.base/unix/native/libnio/ch/FileDispatcherImpl.c

```diff
@@ -28,8 +28,10 @@
 #include <fcntl.h>
 #include <sys/uio.h>
 #include <unistd.h>
+#ifdef MACOSX
 #include <sys/mount.h>
 #include <sys/param.h>
+#endif
```

Comment:
The macOS specific includes are #ifdef _ALLBSD_SOURCE in this file so we can move them there (sorry, there is inconsistency in this source files as to whether _ALLBSD_SOURCE or MACOSX is used, we need to clean this up sometimes).